packages/fast-components-styles-msft/src/slider/index.ts

```diff
@@ -1,264 +1,304 @@
-import { DesignSystem, withDesignSystemDefaults } from "../design-system";
-import { ComponentStyles, ComponentStyleSheet } from "@microsoft/fast-jss-manager";
 import { SliderClassNameContract } from "@microsoft/fast-components-class-name-contracts-msft";
-import { elevation, ElevationMultiplier } from "../utilities/elevation";
+import { ComponentStyles } from "@microsoft/fast-jss-manager";
+import { applyFocusVisible, format, toPx, add } from "@microsoft/fast-jss-utilities";
+import { DesignSystem } from "../design-system";
+import { applyCornerRadius } from "../utilities/border";
 import {
+    neutralFocus,
+    neutralForegroundActive,
     neutralForegroundHint,
     neutralForegroundHover,
     neutralForegroundRest,
     neutralOutlineRest,
 } from "../utilities/color";
+import { applyCursorPointer } from "../utilities/cursor";
+import { heightNumber } from "../utilities/density";
+import {
+    backgroundColor,
+    designUnit,
+    focusOutlineWidth,
+} from "../utilities/design-system";
+import { applyDisabledState } from "../utilities/disabled";
+import { applyElevation, ElevationMultiplier } from "../utilities/elevation";
+
+function thumbSize(config: DesignSystem): string {
```

Comment:
if you create the `thumbSize` as a const, you can get a bit more code-sharing here:

```
const thumbSizeValue: DesignSystemResolver<number> = add(divide(heightNumber(), 2) designUnit);
const thumbSize: DesignSystemResolver<string> = toPx(thumbSizeValue);
const halfThumbSize: DesignSystemResolver<string> = toPx(divide(thumbSizeValue, 2));
```